tutorials/module3-tasks/m3c_continuous_normalizing_flows.ipynb

```diff
@@ -266,8 +280,8 @@
     "    def training_step(self, batch, batch_idx):\n",
     "        self.iters += 1\n",
     "        x, _ = batch \n",
-    "        xtrJ = self.model(x)  \n",
-    "        logprob = prior.log_prob(xtrJ[:,1:]).to(x) - xtrJ[:,0] # logp(z_S) = logp(z_0) - \\int_0^S trJ\n",
+    "        t, xtrJ = self.model(x)\n",
+    "        logprob = prior.log_prob(xtrJ[1, :,1:]).to(x) - xtrJ[1, :,0] # logp(z_S) = logp(z_0) - \\int_0^S trJ\n",
```

Comment:
Could also use `-1` as the index here in case the trajectory/`s_span`/`t_span` is longer